### PR TITLE
wallabag: 2.5.1 -> 2.5.2

### PR DESCRIPTION
**Pull request by mweinelt**
https://github.com/wallabag/wallabag/releases/tag/2.5.2

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Comment by zakame** [comment]
Result of `nixpkgs-review pr 197111` run on aarch64-darwin [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>wallabag</li>
  </ul>
</details>

File: pkgs/servers/web-apps/wallabag/default.nix [hunk 1]
```diff
@@ -15,15 +15,18 @@
...
-    url = "https://static.wallabag.org/releases/wallabag-release-${version}.tar.gz";
-    hash = "sha256-vurjWI5Sh/SFPtxd5cHaaw7edcAzNub/duhOUF+Wshk=";
+    urls = [
+      "https://static.wallabag.org/releases/wallabag-release-${version}.tar.gz"
+      "https://github.com/wallabag/wallabag/releases/download/${version}/wallabag-${version}.tar.gz"
+    ];
+    hash = "sha256-Q989SorGPm3KBuQhGAinYU6HGIa9RrhtRPvwGALU6jk=";
   };
 
   patches = [
```
Comment (jtojnar) [inline pkgs/servers/web-apps/wallabag/default.nix:32]: Might be useful to pick https://github.com/jtojnar/nixfiles/blob/eaf8a54818f4b90a2d48f9fbd6f7aef97f08fdd0/hosts/azazel/ogion.cz/bag/default.nix#L9-L15 to fix sending e-mails.

**Review by zakame — APPROVED** [review]

**Review by jtojnar — APPROVED** [review]
Seems to work.

File: pkgs/servers/web-apps/wallabag/default.nix (hunk 1 above)
Comment (mweinelt) [reply pkgs/servers/web-apps/wallabag/default.nix:32]: Feel free to push it.

Comment (jtojnar) [reply pkgs/servers/web-apps/wallabag/default.nix:33]: Done.

**Comment by mweinelt** [comment]
Typo in the commit message.

**Pull request merged**